src/inspect_ai/analysis/beta/_prepare/model_info.py

```diff
@@ -0,0 +1,44 @@
+from typing import Dict
+
+import pandas as pd
+
+from inspect_ai.analysis.beta._prepare.model_data.model_data import (
+    ModelInfo,
+    read_model_info,
+)
+from inspect_ai.analysis.beta._prepare.operation import Operation
+
+
+def model_info(
+    model_info: Dict[str, ModelInfo] | None = None,
+) -> Operation:
+    # Read built in model info
+    builtin_model_info = read_model_info()
+
+    # Merge with user provided model info
+    resolved_model_info = builtin_model_info | (model_info or {})
+
+    def transform(df: pd.DataFrame) -> pd.DataFrame:
+        # Add columns from ModelInfo for each row based on the 'model' column
+        fields = [
+            "family_display_name",
```

Comment:
I think we need to preface these with `model_` so they are coherent in a larger data frame. Maybe `model_family` and `model_display_name` ?